catapult-sdk/src/utils/convert.js

```diff
@@ -213,6 +213,22 @@ const convert = {
 			throw Error(`input '${input}' is out of range`);
 
 		return (input & 0xFFFFFFFF) >>> 0;
+	},
+
+	/**
+     * Reversed convertion hex string to a uint8 array.
+     * @param {string} input A hex encoded string.
+     * @returns {Uint8Array} A uint8 array corresponding to the input.
+     */
+	hexToUint8Reverse: input => {
```

Comment:
`hexToUint8(input).reverse()` (i'm actually not sure if you need a separate function for this)

Reply:
Yeah, I agree, actually the idea is come from the old sdk.
I will remove it 👍🏼